scripts/test-turbo-modules.sh

```diff
@@ -174,6 +174,11 @@ create_library() {
     rm -rf "$base" || error "Failed to remove existing directory $base"
   fi
 
+  local type="turbo-module"
+  if [[ "$BOB_VERSION" != "0.45.1" && "$BOB_VERSION" = "`echo -e "0.45.1\n$BOB_VERSION" | sort -V | head -n1`" ]]; then
```

Comment:
This is probably fine for now, but this breaks once builder-bob v0.100.0 ships 😊

If this was TypeScript we could have used the `server` package to do this reliably, if you want to make this more robust you could also add it as a dev-dependency and call into `node` here.

Reply:
I'm going to delegate my review approval power to @kraenhansen who seems to be better placed to approve or disapprove.

I would suggest, however, _not_ translating this script into TS/JS, but do it in a different PR.

Reply:
You're right - I didn't realize [the meaning of that](https://www.gnu.org/software/coreutils/manual/html_node/Version-sort-overview.html) 👍